### PR TITLE
Issue with transferring money in "simpletest.js"

### PR DESCRIPTION
**Pull request by foodman47**
### Issue
-  error when trying the transfer method for the first time

### Steps to Reproduce
- benchmark "benchmark/simple/transfer.js" in "benchmark/simple/config.yaml" withtout the fix
   => it will output a recurring error.

### CODE REVIEW

File: packages/caliper-application/benchmark/simple/transfer.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+/**
```
Comment (aklenik) [inline packages/caliper-application/benchmark/simple/transfer.js]: Please use a license header like in https://github.com/hyperledger/caliper/blob/master/packages/caliper-fabric-ccp/lib/fabric.js

File: packages/caliper-application/benchmark/simple/transfer.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    } else {
+        args = {
+            'verb': 'transfer',
+            'account1': account1,
+            'account2': account2,
+            'money': initMoney.toString()
+        };
+    }
+
+    return bc.invokeSmartContract(contx, 'account', 'v0', args, 10);
```
Comment (aklenik) [inline packages/caliper-application/benchmark/simple/transfer.js]: account => simple
(The contract id is `simple` )

Comment (foodman47) [reply packages/caliper-application/benchmark/simple/transfer.js]: Thank you for reviewing my code! I think now it should be all good.

**Comment by aklenik** [comment]
@foodman47 Thanks for the quick fix and for your first contribution 👍

**Pull request merged**